### DISCUSSION TITLE
Ability to properly display Panoramic images

### DISCUSSION THREAD

**Discussion by SpinDreams** [dimsemenov/PhotoSwipe#2120] [Ideas]
Would be great if we could show 360 panoramic images in a gallery (Pan/zoom etc), maybe automatic detection or with an html attribute so you could have a mix of normal and pano images

**Comment by acwolff** [comment]
See these [360 cylindrical panoramas](https://andrewolff.jalbum.net/Panorama_PS/index.html) and these [360 spherical panoramas](https://andrewolff.jalbum.net/PS_SpherePan/).

**Reply by SpinDreams** [reply]
Cool. just what I am looking for, Is that custom coded extension or part of the library? I am using PS as part of Bricks Builder

**Reply by SpinDreams** [reply]
Okay so I read the rest of the page and see you are implementing Photo Sphere to display the panos in conjunction with PhotoSwipe, nice.

**Comment by acwolff** [comment]
Both panorama types are not in the Photoswipe  library, but implemented  with special libraries like the Photo Sphere Viewer of Jérémy Heleine
